orchagent/fdborch.cpp

```diff
@@ -41,6 +46,7 @@ FdbOrch::FdbOrch(DBConnector* applDbConnector, vector<table_name_with_pri_t> app
     /* Add FDB notifications support from ASIC */
     DBConnector *notificationsDb = new DBConnector("ASIC_DB", 0);
     m_fdbNotificationConsumer = new swss::NotificationConsumer(notificationsDb, "NOTIFICATIONS");
+
```

Comment:
Remove extra line. Please keep the previous code intact and fix such line additions

Reply:
Thanks,Done.